lib/src/components/NxSidebarNavigation/NxSidebarNavitation.scss

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-width__open: 250px;
+$nx-sidebar-width__closed: 69px;
+$nx-sidebar-logo-height: 33px;
+$nx-sidebar-nav-background-color: $nx-blue-1000;
+$nx-sidebar-nav-text-color: $nx-white;
+$nx-sidebar-nav-help-color: $nx-grey-50;
+$nx-sidebar-nav-hover-color: $nx-indigo-600;
+$nx-sidebar-nav-selected-color: $nx-indigo-800;
+$nx-sidebar-separator-color: $nx-grey-900;
+
+
+.nx-sidebar-nav {
+  @include container-vertical;
+
+  box-sizing: border-box;
+  background-color: $nx-sidebar-nav-background-color;
+  color: $nx-sidebar-nav-text-color;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  min-width: 250px;
+  padding: 0;
+  padding-top: $nx-spacing-l;
+  position: relative;
+  // transition: all 0.5s ease;
+}
+
+.nx-sidebar-nav--product-info {
+  margin: $nx-spacing-sm $nx-spacing-l 0;
+  height: $nx-sidebar-logo-height;
+}
+
+.nx-sidebar-nav--product-name {
+  display: inline-block;
+  font-size: 16px;
+  line-height: $nx-sidebar-logo-height;
+  vertical-align: top;
+}
+
+.nx-sidebar-nav--logo {
+  height: $nx-sidebar-logo-height;
+  margin-right: $nx-spacing-xs;
+  width: 30px;
+}
+
+.nx-sidebar-nav--icon {
```

Comment:
This class is poorly named.  It should be an element not a modifier, and it is confusing because you use it not on icons, but on specific buttons (that only contain icons).

Reply:
> This class is poorly named. It should be an element not a modifier, and it is confusing because you use it not on icons, but on specific buttons (that only contain icons).

I got rid of it entirely in 232611c992543e74499f2aab8381c27efdf6e960